iotdevices/src/main/java/rest/api/DeviceService.java

```diff
@@ -106,34 +107,36 @@ public Response submitFeedback(@HeaderParam("feedback") String feedback,
         } catch (NotFoundException e) {
             return Response.status(404).entity( "No such user" + userid).build();
         } catch (Exception e) {
-            return Response.status(404).entity("Unknown error").build();
+            return Response.status(404).entity(e).build();
```

Comment:
Perhaps not the best to just send the exception, but isn't important now

Reply:
agreed, I will remove it. I had it for debugging purposes, but forgott before committing

Reply:
:gb: